### PR TITLE
Export testutils package

### PR DESCRIPTION
**Pull request by warber**
This PR exports the testutils package.

### CODE REVIEW

File: api/clients/buckets/bucket_test.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/clients/buckets"
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
-	"github.com/dynatrace/dynatrace-configuration-as-code-core/internal/testutils"
+	testutils2 "github.com/dynatrace/dynatrace-configuration-as-code-core/testutils"
```
Comment (UnseenWizzard) [inline api/clients/buckets/bucket_test.go:22]: no need for `testutils2` here as well I think?

File: clients/buckets/bucket_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api"
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/clients/buckets"
-	"github.com/dynatrace/dynatrace-configuration-as-code-core/internal/testutils"
+	testutils2 "github.com/dynatrace/dynatrace-configuration-as-code-core/testutils"
```
Comment (UnseenWizzard) [inline clients/buckets/bucket_test.go:23]: testutils2

**Comment by UnseenWizzard** [comment]
I assume you're using this for testing the account client in monaco? 
Exporting for that is fine by me, but I think it might be another argument for creating a "smart" client for the account API in core-

**Review by UnseenWizzard — DISMISSED** [review]

File: api/clients/automation/automation_test.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ package automation_test
 import (
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/clients/automation"
 	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
-	"github.com/dynatrace/dynatrace-configuration-as-code-core/internal/testutils"
+	testutils2 "github.com/dynatrace/dynatrace-configuration-as-code-core/testutils"
```
[the remark this answers is not in the document]
Comment (warber) [reply api/clients/automation/automation_test.go]: yes :D

**Review by UnseenWizzard — APPROVED** [review]

**Pull request merged**